modules/tensor_mechanics/doc/content/syntax/BCs/InclinedNoDisplacementBC/index.md

```diff
@@ -0,0 +1,24 @@
+# Inclined No Displacement Boundary Condition Action System
+
+!syntax description /BCs/InclinedNoDisplacementBC/InclinedNoDisplacementBCAction
+
+## Description
+
+The InclinedNoDisplacementBCAction Action, given in the input file as simply `InclinedNoDisplacementBC`, is designed to simplify the input file when several variables have the same inclined no displacement boundary condition [Inclined no displacement boundary condition](PenaltyInclinedNoDisplacementBC.md)  applied in the normal component.
+
+The Pressure Action, given in the input file as simply `Pressure`, is designed to simplify the input file when several variables have the same pressure boundary condition magnitude applied in the normal component.
```

Comment:
I think you want to remove these two lines.

Reply:
done